latex2sympy.py

```diff
@@ -104,16 +104,37 @@ def process_sympy(self):
     def pre_process_latex(self, latex):
         '''
         pre-processing for issues the parser cannot handle
+
+        find any single char sup/sub and wrap them in "{}"
+        e.g. `4^26^2` => `4^{2}6^{2}`
+        e.g. `x_22` => `x_{2}2`
```

Comment:
@builtbywill Do we not allow multi-digit subscripts? Otherwise how do we distinguish between `x_{2}2` and `x_{22}`

Reply:
in other words, the front-end would have already wrapped the whole thing as `x_{22}` before it hits this point

the only time in the past it was _not_ wrapped was when it was typed in the front-end as a single char, like the examples above